src/components/delete_account/index.js

```diff
@@ -1,53 +1,51 @@
 import React from 'react';
-import {Trans, withTranslation} from 'react-i18next';
-import IrmaButton from '../../helpers/irma_button';
-import IrmaModal from '../../helpers/irma_modal';
-import {connect} from 'react-redux';
+import { Trans, withTranslation } from 'react-i18next';
+import IrmaButton from '../../widgets/irma_button';
+import IrmaModal from '../../widgets/irma_modal';
+import './index.scss';
```

Comment:
This file does not exist.

Reply:
Sharp, little mistake in resolving the merge conflict.